platform/android/app/src/test/kotlin/coop/polypoly/polypod/updatenotification/UpdateNotificationTest.kt

```diff
@@ -1,88 +0,0 @@
-package coop.polypoly.polypod.updatenotification
```

Comment:
Are you not going to keep the test? Move it somewhere else?

Reply:
It moved to core - that's where the update notification logic is implemented now, so the unit tests are also there. Previously it was duplicated.

Reply:
So this removes a duplicated file, right?